discojs/src/index.ts

```diff
@@ -1,7 +1,5 @@
 export * as data from './dataset/index.js'
 export * as serialization from './serialization/index.js'
-export { Encoded as EncodedModel } from './serialization/model.js'
-export { Encoded as EncodedWeights } from './serialization/weights.js'
```

Comment:
correct, everything is `Uint8Array` now. previously, it was kinda the case, one being `number[]` (but with only 0-255 values), the other `Uint8Array`. I can reintroduce the two if you prefer but a `isEncoded{Model,Weights}` function can't as it would require actually decoding the whole array to differenciate between Model & Weights.

I actually kept the `EncodedModel` in server/task_set to make it more explicit, which can help where the meaning of "Encoded" isn't obvious.